src/operator/numpy/np_matrix_op.cc

```diff
@@ -464,5 +464,44 @@ NNVM_REGISTER_OP(_np_squeeze)
 .add_argument("a", "NDArray-or-Symbol[]", "data to squeeze")
 .add_arguments(SqueezeParam::__FIELDS__());
 
+NNVM_REGISTER_OP(_npi_hsplit)
+    .add_alias("_npi_hsplit")
```

Comment:
If you already registered it as `_npi_split` then you do not need an extra `_npi_split` alias